### ISSUE TITLE
[BUG] PaymentSheet primaryButtonColor blinking or flash when set color other than blue

### ISSUE THREAD

**Issue by unxavi** [stripe/stripe-ios#1936]
## Summary
Showing a PaymentSheet with a configuration of primaryButtonColor other than blue, makes it show and  blink or flash to blue for a short interval in SwiftUI (I'm not sure about UIKit)

Video:
https://user-images.githubusercontent.com/3817679/157838848-e6357d56-f486-40c6-a2c6-2c4db34ffb81.mov

## Code to reproduce

```
var configuration = PaymentSheet.Configuration()
configuration.applePay = .init(
    merchantId: applePayMerchantId,
    merchantCountryCode: "US"
)

configuration.customer = .init(
    id: stripeSubscriptionIntent.customerId,
    ephemeralKeySecret: stripeSubscriptionIntent.ephemeralKeySecret
)
configuration.primaryButtonColor = .purple
paymentSheet = PaymentSheet(
    paymentIntentClientSecret: stripeSubscriptionIntent.paymentIntentSecret,
    configuration: configuration
)
```

## iOS version
15.3.1

## Installation method
SPM

## SDK version
21.12.0

**Comment by porter-stripe** [comment]
Hi @unxavi! That is some odd behavior, I was able to reproduce this issue as well. I have submitted a fix to our project and it will be included in the next release. I'll update this issue when the fix has shipped!

**Comment by porter-stripe** [comment]
This is fixed in [21.13.0](https://github.com/stripe/stripe-ios/releases/tag/21.13.0)!
